iwyu_path_util.cc

```diff
@@ -83,8 +83,7 @@ string Basename(const string& path) {
 
 string GetCanonicalName(string file_path) {
   // Get rid of any <> and "" in case file_path is really an #include line.
-  StripLeft(&file_path, "\"") || StripLeft(&file_path, "<");
-  StripRight(&file_path, "\"") || StripRight(&file_path, ">");
+  StripQuotes(&file_path);
```

Comment:
Actually, now that I look at it, I can't find a single place where this is called with a quoted path. So let's replace `StripQuotes` with:

    CHECK_(!IsQuotedInclude(file_path));

and remove `StripQuotes` entirely.